tools/perf/ucc_pt_coll_gatherv.cc

```diff
@@ -0,0 +1,94 @@
+#include "ucc_pt_coll.h"
+#include "ucc_perftest.h"
+#include <ucc/api/ucc.h>
+#include <utils/ucc_math.h>
+#include <utils/ucc_coll_utils.h>
+
+ucc_pt_coll_gatherv::ucc_pt_coll_gatherv(ucc_datatype_t dt,
+                         ucc_memory_type mt, bool is_inplace,
+                         ucc_pt_comm *communicator) : ucc_pt_coll(communicator)
+
+{
+    has_inplace_   = true;
+    has_reduction_ = false;
+    has_range_     = true;
+    has_bw_        = true;
```

Comment:
should be false because get_bw() is not implemented, what gatherv prints in full output